packages/beacon-node/src/metrics/metrics/beacon.ts

```diff
@@ -71,6 +72,82 @@ export function createBeaconMetrics(register: RegistryMetricCreator) {
       help: "Current number of pending partial withdrawals",
     }),
 
+    eip7805: {
+      validInclusionListByteSize: register.counter({
+        name: "beacon_valid_inclusion_lists_size_bytes_total",
+        help: "Byte size of the inclusion list",
+      }),
+      invalidInclusionListByteSize: register.counter({
+        name: "beacon_invalid_inclusion_lists_size_bytes_total",
+        help: "Byte size of the inclusion list",
+      }),
+      inclusionListSeen: register.counter<{source: InclusionListSource}>({
+        name: "beacon_inclusion_lists_seen_total",
+        help: "Total number of seen inclusion lists (gossip, api)",
+        labelNames: ["source"],
+      }),
+      inclusionListValid: register.counter<{source: InclusionListSource}>({
+        name: "beacon_valid_inclusion_lists_total",
+        help: "Total number of valid inclusion lists",
+        labelNames: ["source"],
+      }),
+      inclusionListInvalid: register.counter<{source: InclusionListSource; reason: InvalidInclusionListReason}>({
+        name: "beacon_invalid_inclusion_lists_total",
+        help: "Total number of invalid inclusion lists",
+        labelNames: ["source", "reason"],
+      }),
+      inclusionListValidationTime: register.histogram<{source: InclusionListSource}>({
+        name: "beacon_inclusion_lists_validation_time_seconds",
+        help: "Time taken to validate inclusion list",
+        buckets: [0.01, 0.05, 0.1, 0.5, 1, 2, 5],
```

Comment:
I think we can remove 5 second bucket, validation is really cheap, could even get rid of 1 second bucket

```suggestion
        buckets: [0.001, 0.005, 0.01, 0.05, 0.1, 0.5, 1],
```

just a suggestions but I would rather add more small buckets